config/dispatcher/dispatcher.ex

```diff
@@ -84,7 +93,7 @@ defmodule Dispatcher do
   #################################################################
   # Reports
   #################################################################
-  match "/reports/*path" do
+  match "/reports/*path",  %{ accept: [:json] } do
```

Comment:
Without this the `/reports` endpoint of the dashboard would display the json-api response since it's defined before the frontend. Most apps add these accept headers for most match statements, but that felt out of scope for this PR.